### PR TITLE
add `zh-TW` translations.

### PR DESCRIPTION
**Pull request by memorysaver**
## Implementation
- add zh-TW languages templates for FAQ and homepage.
- translate to traditional Chinese language
- remove some unnecessary spaces by lint tool

## Todo
According to the discussion on Keybase, the detail about  the required `minimum hard drive space` for Chia at FAQ section needs to be refined after the changes in English version. 

Moreover, once the PR is approved. `Simplified Chinese` will be in the plan for sure.

### CODE REVIEW

**Comment by ashadle** [comment]
@memorysaver what question are you referring to on the FAQ page about the required space for Chia?

**Comment by memorysaver** [comment]
@ashadle  I think edec13e has been updated for the required minimum space for Chia. I think I can update this PR after #19 is merged.

**Comment by dkackman** [comment]
Just as a heads up, since you have a fully specified LCID string, locales.faq.js will need to be updated to do something like:

Try full faq.[LCID].md
Try just language code faq.[langacode].md
Default to faq.md. (english)

Right now it expects just the 2 character language code, not the full LCID. Easy enough to change. If you don't get to it I will hopefully have a chance later this week.

**Comment by memorysaver** [comment]
@dkackman  I tried LCID string as 'zh'. However, it won't work instead of `zh-tw`. I understand that the instructions mentioned it expects only 2 character language code. Also, I think Chinese language won't work for only 2 character situation. `zh-tw` and `zh-cn` may use different vocabulary and Chinese character, but two type of vocabulary and character are understandable for both native speaker. 

Do you still recommend with only 2 character convention? I can also take a look of the code to see why it won't work.

**Comment by dkackman** [comment]
> Do you still recommend with only 2 character convention? 

No. I think the current 2 character limitation should be fixed so that it works for dialects with significant differences, like Chinese. I will have time to get to that later today.

**Comment by dkackman** [comment]
I just added a pull request to your feature branch (I think - never tried to amend someone else's PR on GitHub) with the necessary script change to support loading faq.zh-TW.md etc.

**Comment by memorysaver** [comment]
thanks for the help !!!

File: webroot/faq/locales/faq.zh-TW.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
++ <b>Q: 耕作的意思是?</b>
+
+A: Chia 耕作就像挖礦一樣創造區塊. 耕種透過完成您的硬碟可用空間的一個快照 (seeding)，根據您想要提供給 Chia 的耕作空間轉換為獲取獎賞的機率，提供越多的空間就能得到越多得獎的機會。雖然如此，此設計不會激勵使用者購買硬碟來耕作。 因為 Chia 是利用擁有者目前閒置的硬碟空間來耕作，並不需要擁有者付出更多的資源。 然而世上有非常大量的儲存空間，對於一般使用者來說獲得獎勵的機會幾乎微乎其微，在考慮到 Chia 只需要使用少量的能源，頻寬，以及閒置的空間，使用者並不會在意獎勵是少的。同樣的，Chia 也不會激勵礦池，獨立挖礦，或使用 ASIC 礦機的行為。
+
+
+
++ <b>Q: Chia 與比特幣最大的不同?</b>
+
+A: Chia 取代挖礦利用計算能力而改用硬碟空間來工作，讓使用者更容易加入，大量的降低為了保護網路而產生的能源成本及浪費。 Chia 是反對耕作中心化的。 Chia 有許多基本的進步： 所有的交易都是 BLS signatures 來提供更好的智慧交易支援，並且修復許多像是 timewarp 這樣的臭蟲。 目標是讓 hard fork 更加不可能發生，所以治理將會來自於使用者的選擇以及如同比特幣般去中心化的激勵。
```
Comment (darkmagic666) [inline webroot/faq/locales/faq.zh-TW.md]: hard fork 硬分岔

**Comment by ashadle** [comment]
@memorysaver would you mind updating your code so it can be merged?

**Comment by dkackman** [comment]
@memorysaver looks like this has a conflicting change in the Javascript. I can do a merge this weekend.

**Comment by memorysaver** [comment]
sure thing please ! I am sorry that it's quite busy in Taiwan before Chinese New Year. I'll get back to work on this when I am available!

**Comment by detailyang** [comment]
@memorysaver @ashadle 
Hello.
I'm pleasure to review ` Simplified Chinese` PR, if this PR about `zh-TW` is approved:)

**Comment by ashadle** [comment]
@memorysaver can you resolve the conflicts so I can merge this? Thank you!

**Comment by ashadle** [comment]
Hi There, thank you so much for submitting this PR. We are rebranding the website and will have updated copy so I'm going to close out this ticket. Thank you!

**Pull request closed without merging**